### PR TITLE
3.3.8.4 fix release changes

### PR DESCRIPTION
**Pull request by zebra-lucky**
- Trezor/KeepKey plugins: fix signing of CbTx outputs (#100)
- fix Disable PrivateSend and PrivateSend UI for HW wallets (unsupported)
- allow mixing amounts below keep amount, other ps/ps gui fixes
- add PPA for Focal Fossa (Ubuntu 20.04)
- android arm64-v8a apk build
- update Tor Proxy in win builds to 0.4.2.6

### CODE REVIEW

File: electrum_dash/gui/qt/address_list.py [hunk 1]
```diff
@@ -75,6 +78,11 @@ def run(self):
             except Exception as e:
                 self.model.logger.error(f'GetDataThread error: {str(e)}')
 
+    def stop(self):
```
Comment (jardev) [inline electrum_dash/gui/qt/address_list.py:81]: Code duplication. Exactly the same code found in `history_list.py` and `utxo_list.py`.

File: electrum_dash/gui/qt/history_list.py [hunk 2]
```diff
@@ -118,6 +121,11 @@ def run(self):
             except Exception as e:
                 self.model.logger.error(f'GetDataThread error: {str(e)}')
 
+    def stop(self):
```
Comment (jardev) [inline electrum_dash/gui/qt/history_list.py:124]: Code duplication. Exactly the same code also found in `address_list.py` and `utxo_list.py`.

File: electrum_dash/gui/qt/utxo_list.py [hunk 3]
```diff
@@ -72,6 +75,11 @@ def run(self):
             except Exception as e:
                 self.model.logger.error(f'GetDataThread error: {str(e)}')
 
+    def stop(self):
```
Comment (jardev) [inline electrum_dash/gui/qt/utxo_list.py:78]: Code duplication. Exactly the same code also found in `address_list.py` and `history_list.py`.

File: electrum_dash/gui/qt/history_list.py (hunk 2 above)
Comment (zebra-lucky) [reply electrum_dash/gui/qt/history_list.py]: fixed in a7e7d19

File: electrum_dash/gui/qt/utxo_list.py (hunk 3 above)
Comment (zebra-lucky) [reply electrum_dash/gui/qt/utxo_list.py]: fixed in a7e7d19

File: electrum_dash/gui/qt/address_list.py (hunk 1 above)
Comment (zebra-lucky) [reply electrum_dash/gui/qt/address_list.py]: fixed in a7e7d19

**Pull request merged**